pkg/server/server.go

```diff
@@ -767,10 +767,10 @@ func NewServer(cfg Config, stopper *stop.Stopper) (*Server, error) {
 		SnapshotSendLimit:        cfg.SnapshotSendLimit,
 		RangeLogWriter:           rangeLogWriter,
 	}
-
 	if storeTestingKnobs := cfg.TestingKnobs.Store; storeTestingKnobs != nil {
 		storeCfg.TestingKnobs = *storeTestingKnobs.(*kvserver.StoreTestingKnobs)
 	}
+	storeCfg.SetDefaults()
```

Comment:
Could imagine this would break a few tests. I guess it hasn't, since CI looks green. Nice!

Reply:
Yeah, most tests use `testStoreConfig()` which already calls `SetDefaults()`.